libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -782,6 +788,24 @@ void AP_OSD_Screen::draw_temp(uint8_t x, uint8_t y)
     backend->write(x, y, false, "%3d%c", (int)u_scale(TEMPERATURE, tmp), u_icon(TEMPERATURE));
 }
 
+void AP_OSD_Screen::draw_dist(uint8_t x, uint8_t y)
+{
+    uint32_t now = AP_HAL::millis();
+    uint32_t delta = now - osd->last_update_ms;
+    AP_AHRS &ahrs = AP::ahrs();
+    Vector2f v = ahrs.groundspeed_vector();
+    float speed = v.length();
+    if (delta > 500) {
+        if (speed < 2.0) speed = 0.0;
```

Comment:
Is this needed?

Reply:
yes...it prevents distance being accumulated while sitting on the ground and ground speed bumps around randomly from 0 up to 2....if IS_FLYING was available to the library functions, that could be the qualifier, but its not apparently...alternatively, I could add an additional storage variable to track flightime and qualify it based on increasing flight time...this was the simplest

Reply:
Ok. It make sense.